test/cases/load_data/import_data.result

```diff
@@ -0,0 +1,320 @@
+drop table if exists t1;
```

Comment:
we've removed the `test/cases` folder, so u don't need provide standalone cases anymore